### ISSUE TITLE
cp: can't stat '/flash/SYSTEM' : No such file or directory

### ISSUE THREAD

**Issue by gordielachance** [OpenELEC/OpenELEC.tv#4933]
Hi guys,

It's really easy to install OpenElec on a fresh drive, but what a pain to be able to make a multi boot with it :/  
Too bad there is no way to select an existing "system" and "storage" partition when installing it.

Well, I made several trials to make it work along Mint; but none where sucessfull.

My last attempt was to install OpenElec and clone its partitions; then to format the disk, install Mint; and restore the OpenElec partitions.  I thought it was a good idea, but that does not work at all :/

I have this error when booting OpenElec : 

> cp: can't stat '/flash/SYSTEM' : No such file or directory

I saw a lot of people are getting this error too, but I didn't find any answer.
I don't understand either what is "flash".
[This post that is related to it on Github](https://github.com/OpenELEC/OpenELEC.tv/issues/2809), but that is too technical for me, I don't get it.
Could someone help ?

Thanks !

**Comment by gordielachance** [comment]
.. Or since I recreated the partitions (in Mint); could this be related to file permissions ?

**Comment by sraue** [comment]
please post infos about your partition scheme, bootloaderconfig and partition names

**Comment by vpeter4** [comment]
gordielachance, multiboot is simple if you understand how oe is booted. You need to set the label for partitions correctly or change boot=/disk= in syslinux.cfg to point to right partitions. Labels are easy.

**Comment by gordielachance** [comment]
Hi guys, thanks for your suggestions.  

I finally managed to install OpenElec the way I wanted to : installing Mint first, then creating openElec partitions (system & storage), then copying the system files to the first partition and updated the grub2, [following the wiki instructions](http://wiki.openelec.tv/index.php/Dual_Boot), and it worked.

However, one remark : it didn't work when I used labels for the partitions in _40_custom_ file for grub2 (I had an error, something "unknow file system"); but it worked with UUIDs for the disks.  

I'm not sure why.  Maybe the case or something.  Anyway, installing it that way is possible then !  Thanks.
